arch/arm64/src/common/arm64_gicv3.c

```diff
@@ -789,6 +789,28 @@ uint64_t * arm64_decodefiq(uint64_t * regs)
 
   irq = arm64_gic_get_active_fiq();
 
+#ifdef CONFIG_ARCH_BOOT_EL3
+  /* FIQ is group0 interrupt */
+
+  if (irq == 1021)
```

Comment:
Please use a #define instead of hardcoding the value 1021

Reply:
Sure, it kind of follows the logic seen in the file; eg. 1023:
DEBUGASSERT(irq < NR_IRQS || irq == 1023);
Probably rework both 1021 and 1023?

Reply:
Being just fixed, thanks!